packages/account/src/Constants/connected-apps-config.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Localize } from '@deriv/translations';
+
+export const CONNECTED_APPS_INFO_BULLETS = [
+    {
+        key: 'key_1',
```

Comment:
Suggestion: can we simply say `key: 1` or `index: 1` rather than explicitly appending `key_1`, `key_2`?